server/ERNI.PBA.Server.DataAccess/Repository/InvoiceImageRepository.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using ERNI.PBA.Server.DataAccess.Model;
+using Microsoft.EntityFrameworkCore;
+
+namespace ERNI.PBA.Server.DataAccess.Repository
+{
+    public class InvoiceImageRepository : IInvoiceImageRepository
+    {
+        private readonly DatabaseContext _context;
+
+        public InvoiceImageRepository(DatabaseContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<Tuple<int,string>[]> GetInvoiceImagesNameId(int requestId, CancellationToken cancellationToken)
```

Comment:
try to use names tuple here

Reply:
Named tuples dont work well with EF Linq selec . Double conversion is needed